packages/11ty/_plugins/vite/index.js

```diff
@@ -0,0 +1,90 @@
+const copy = require('rollup-plugin-copy')
+const path = require('path')
+const EleventyVitePlugin = require('@11ty/eleventy-plugin-vite')
+
+module.exports = function(eleventyConfig, { inputDir, outputDir, globalData }) {
+  /**
+   * Use Vite to bundle JavaScript
+   * @see https://github.com/11ty/eleventy-plugin-vite
+   *
+   * Runs Vite as Middleware in the Eleventy Dev Server
+   * Runs Vite build to postprocess the Eleventy build output
+   */
+  const { pathname } = globalData.publication
+  const pathResolutionAliases = pathname === '/' ? [] : [{ find: pathname, replacement: '/' }]
```

Comment:
Moving this expression to line 28 to set the `alias` property directly may be more explicit.